### PR TITLE
'DuckDuckGo' trademark test

### PR DESCRIPTION
**Pull request by jbarrett**
Cc: @andrey-p 

CI should show us some interesting cases.

### CODE REVIEW

**Comment by jbarrett** [comment]
A fair chunk of these are typos or capitalisation differences.

**Comment by akanshgulati** [comment]
Exactly, I was about to ask the same question regarding the correct way to write Duckduckgo. I think, now I know the correct spelling for it. `DuckDuckGo`.

**Comment by dildiwana** [comment]
@jbarrett  How are we planning to update the trademark for different languages as when I tried with `Arabic` language, I was able to update the string using Google Translate but still not sure if that string will make any meaning.

**Comment by akanshgulati** [comment]
@jbarrett How are we planning to update the trademark for different languages as when I tried with Arabic language, I was able to update the string using Google Translate but still not sure if that string will make any meaning. So, can we somehow just make sure for future PRs only this test run or otherwise would have to look for all languages.

**Comment by jbarrett** [comment]
@akanshgulati Still figuring that part out. Some of these are clearly an easy fix, but others should be left as-is until a native speaker can inform us on what's best.

I was thinking the PO files could have a flag or comment like `#, translated-trademark` which would cause the test to skip them, but let translators know some action is required. That should work for future contributions and also allow some leeway for edge cases (like using [`## no critic`](https://metacpan.org/pod/perlcritic#BENDING-THE-RULES))

I'm taking my own cue from [advice I've found on translation forums](https://www.proz.com/forum/translation_theory_and_practice/123499-translation_of_trademarks.html) as well as what appears to be the industry-standard approach to trade marks:

Screen Shot 2019-11-01 at 10 55 50

Screen Shot 2019-11-01 at 10 58 29

**Comment by akanshgulati** [comment]
@jbarrett Looks fair and apt. We should add the check in such a way that either an entire file is excluded or a few translations are excluded (this will help anyone to make a few changes in a single file and excluding rest). Can you update the test file with the check?

**Comment by jbarrett** [comment]
I added a flag to skip the test for existing translations (with fuzzy included so translators know a review is required).

If you add this flag to the metadata msgid ("") then the whole file will be skipped.

File: locales/hi_IN/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -173,6 +174,7 @@ msgid "Ad is suspicious"
 msgstr "विज्ञापन सन्देहजनक है"
 
 #. From Home page, options to "Add DuckDuckGo" to browser or set as home page.
+#, fuzzy,translated-trademark
 msgid "Add DuckDuckGo"
 msgstr "DuckDuckGo को लगाए"
```
Comment (andrey-p) [inline locales/hi_IN/LC_MESSAGES/duckduckgo.po]: The translations in this file were fixed in a different PR. Looks like we can just drop the `#, fuzzy,translated-trademark` now?

File: locales/bg_BG/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -2533,6 +2534,7 @@ msgctxt "mobile homepage banner"
 msgid "Open %sMenu%s"
 msgstr ""
 
+#, fuzzy,translated-trademark
 msgid "Open %sSettings > Safari > Search Engine%s and select DuckDuckGo!"
 msgstr "Отворете %sНастройки > Safari > Търсачка%s и изберете!"
```
Comment (andrey-p) [inline locales/bg_BG/LC_MESSAGES/duckduckgo.po]: These are kinda weird, it looks like "DuckDuckGo" is missing there. Like, the sentences look like they've had a word chopped off. Hope you don't mind if I jump in and fix it.
